src/NSrtm.Core/Pgm/DataDesciption/PgmDataDescription.cs

```diff
@@ -0,0 +1,52 @@
+﻿namespace NSrtm.Core.Pgm
+{
+    internal struct PgmDataDescription
+    {
+        private readonly double _offset;
+        private readonly double _scale;
+        private readonly int _originLat;
+        private readonly int _originLon;
+        private readonly int _gridGraphWidthPoints;
+        private readonly int _gridGraphHeightPoints;
+        private readonly int _maxValue;
+        private readonly int _preambleLength;
+        private readonly int _numberOfPoints;
+        private readonly double _latitudeIncrement;
+        private readonly double _longitudeIncrement;
+
+        public PgmDataDescription(
+            double offset,
+            double scale,
+            int originLat,
+            int originLon,
+            int gridGraphWidthPoints,
+            int gridGraphHeightPoints,
+            int maxValue,
+            int preambleLength)
+        {
+            _offset = offset;
+            _scale = scale;
+            _originLat = originLat;
+            _originLon = originLon;
+            _gridGraphHeightPoints = gridGraphHeightPoints;
+            _gridGraphWidthPoints = gridGraphWidthPoints;
+            _maxValue = maxValue;
+            _preambleLength = preambleLength;
+            _numberOfPoints = _gridGraphHeightPoints * _gridGraphWidthPoints;
+            _longitudeIncrement = _gridGraphWidthPoints / 360.0;
```

Comment:
So increment is number of values per one degree?

Imho increment is value satisfying:
lat[i] + increment = lat[i+1]..